daemon/execdriver/native/driver.go

```diff
@@ -52,11 +54,33 @@ func NewDriver(root, initPath string) (*driver, error) {
 	if err := apparmor.InstallDefaultProfile(); err != nil {
 		return nil, err
 	}
+
+	// choose cgroup manager
 	cgm := libcontainer.Cgroupfs
 	if systemd.UseSystemd() {
 		cgm = libcontainer.SystemdCgroups
 	}
 
+	// parse the options
+	for _, option := range options {
+		key, val, err := parsers.ParseKeyValueOpt(option)
+		if err != nil {
+			return nil, err
+		}
+		key = strings.ToLower(key)
+		switch key {
+		case "native.cgroupdriver":
+			// override the default if they set options
+			if val == "cgroupfs" || !systemd.UseSystemd() {
+				cgm = libcontainer.Cgroupfs
+			} else {
```

Comment:
it defaults to Systemd only if systems is enabled. You still don't need the `else`, it will default to whatever it's set by default before the option is checked, won't it?

It might need some tests so we can solve this mystery!